kem/tls/tls.go

```diff
@@ -0,0 +1,30 @@
+// Package tls adds helpers to integrate the KEMs supported in Circl into TLS.
+package tls
+
+import (
+	"github.com/cloudflare/circl/kem"
+	"github.com/cloudflare/circl/kem/schemes"
+)
+
+var (
+	allSchemesByCurveID map[uint]kem.Scheme
+)
+
+func init() {
+	allSchemesByCurveID = make(map[uint]kem.Scheme)
+	for _, scheme := range schemes.All() {
+		if tlsScheme, ok := scheme.(TLSScheme); ok {
+			allSchemesByCurveID[tlsScheme.TLSCurveID()] = scheme
+		}
+	}
+}
+
+// Returns scheme with the given TLS CurveID, if any.
+func SchemeByCurveID(id uint) kem.Scheme { return allSchemesByCurveID[id] }
```

Comment:
this function may be moved to `/circl/kem/schemes` which is the place where the register lives, doing so we don't need the `/circl/kem/tls` package, which may be confused with the stdlib package.

Reply:
I used a separate package to be consistent with the pki package, that provides similar functionality but then querying signature schemes by OID. If you’re ok being inconsistent with that, I’ll move it.

Reply:
the issue I'm pointing is that when importing `/circl/kem/tls`, it looks like is a replacement of the `tls` package. what about renaming to `tls_helpers`?
or putting these functions in the `pki` package.